ambari-common/src/main/python/resource_management/libraries/script/script.py

```diff
@@ -791,6 +794,19 @@ def load_available_packages(self):
       Logger.exception("Unable to load available packages")
       self.available_packages_in_repos = []
 
+  def create_component_instance(self):
+    config = self.get_config()
+    mpack_name = get_mpack_name(config)
+    mpack_version = get_mpack_version(config)
+    mpack_instance_name = get_mpack_instance_name(config)
+    service_type = get_service_type(config)
```

Comment:
Lets call this as "module_name" and "get_module_name". From the mpack author perspective they will be more familiar with with mpacks, modules, components rather than stacks, services, components